.github/workflows/socrata_upload.yaml

```diff
@@ -0,0 +1,77 @@
+name: update_socrata_asset
+
+on:
+  workflow_dispatch:
```

Comment:
We'll want to add a cron component as well with some different default values, but I just wanted to get this preliminary build finished up.